cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -64,6 +64,14 @@ spec:
       labels:
         k8s-app: kube-proxy
     spec:
+      initContainers:
+      - name: touch-lock
+        image: {{ .ImageRepository }}/busybox-{{ .Arch }}
```

Comment:
@cmluciano Actually you can use upstream images, but that will add more logic. In this case it seems like the only approach.
So add `{{ .BusyboxImage }}` or similar and set it conditionally:

| GOARCH | Image |
| --- | --- |
| amd64 | busybox |
| arm | armhf/busybox |
| arm64 | aarch64/busybox |
| ppc64le | ppc64le/busybox |
| s390x | s390x/busybox |